src/TLuaInterpreter.cpp

```diff
@@ -12587,10 +12587,10 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
     //QDateTime time = QDateTime::currentDateTime();
     // QString entry = QString("[%1]object:<%2> function:<%3> error:<%4>").arg(time.toString("MMM:dd:yyyy hh-mm-ss")).arg(name).arg(function).arg(e.c_str());
     //mpHost->mErrorLogStream << entry << endl;
-    QColor blue = QColor(0,0,255);
-    QColor green = QColor(0,255,0);
-    QColor red = QColor(255,0,0);
-    QColor black = QColor(0,0,0);
+    QColor blue = QColor(Qt::blue);
+    QColor green = QColor(Qt::green);
+    QColor red = QColor(Qt::red);
+    QColor black = QColor(Qt::black);
```

Comment:
That's not possible to do. The function expects a QColor so the constants don't work, and QColor(<constant>) says expression must be lvalue.